.changelog/unreleased/improvements/ibc-relayer/1971-max-msg-num-min-bound.md

```diff
@@ -0,0 +1,3 @@
+- Ensure `max_msg_num` is always greater than or equal
```

Comment:
Let's delete this file before merging.

Reply:
Isn't that a change worth documenting?

Reply:
But we're not enforcing it to be bigger than 2 anymore. From a user perspective, not much changes.

I guess we can keep the changelog, but then let's adjust the message to be consistent with the implementation. As I understand it, we allow it to be [1, 100] with a default of 30.

https://github.com/informalsystems/ibc-rs/blob/143e7f3dc16b568acb7ade342b05febe1d84183a/relayer/src/config/types.rs#L31-L33

Reply:
Right, my bad!

Reply:
Fixed